lm_eval/tasks/korunsmile.py

```diff
@@ -0,0 +1,71 @@
+"""
+ Korean UnSmile Dataset
+ 
+ Github: https://github.com/smilegate-ai/korean_unsmile_dataset
+"""
+
+import numpy as np
+from lm_eval.base import MultipleChoiceTask
+from lm_eval.metrics import macro_f1_score
+
+_CITATION = """
+@misc{SmilegateAI2022KoreanUnSmileDataset,
+  title         = {Korean UnSmile dataset: Human-annotated Multi-label Korean Hate Speech Dataset},
+  author        = {Seonghyun Kim},
+  year          = {2022},
+  howpublished  = {https://github.com/smilegate-ai/korean_unsmile_dataset},
+}
+"""
+
+class KorUnSmile(MultipleChoiceTask):
+    VERSION = 0
+    DATASET_PATH = "smilegate-ai/kor_unsmile"
+    DATASET_NAME = None
+
+    def has_training_docs(self):
+        return True
+
+    def has_validation_docs(self):
+        return True
+
+    def has_test_docs(self):
+        return False
+
+    def training_docs(self):
+        if self._training_docs is None:
+            self._training_docs = list(map(self._process_doc,self.dataset["train"]))
+        return self._training_docs
+
+    def validation_docs(self):
+        return map(self._process_doc,self.dataset["valid"])
+
```

Comment:
@cardy20 Because `construct_requests` is inherited from `MultipleChoiceTask`, there's no need to define `construct_requests` again. In addition, I modified the overall setting to multi-class classification task that we discussed on previous meeting. Thank you for the helpful review :)